### DISCUSSION TITLE
scikit-image Requirement

### DISCUSSION THREAD

**Discussion by DanielTBaker** [danielreardon/scintools#23] [Q&A] [answered]
I was wondering about the scikit-image installation requirement. I'm having a hard time getting that version installed on my laptop so I just removed the requirement and everything still seems to run.

Was it just for the inpainting?

**Comment by danielreardon** [comment]
Yes that's just for the biharmonic inpainting. I also had issues with that recently, so I'll see if I can get the inpainting to work on the latest versions

**Reply by danielreardon** [reply, marked as answer]
Hi @DanielTBaker I just committed an update to the requirements that gets rid of the scikit-image version. Could you try installing again and see if it works for you?

**Reply by DanielTBaker** [reply]
That seems to have fixed it. Thanks!
